x-pack/plugins/apm/public/services/rest/createCallApmApi.ts

```diff
@@ -4,11 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { HttpSetup } from 'kibana/public';
-import { callApi, FetchOptions } from './callApi';
+import { FetchOptions } from '../../../common/fetch_options';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { APMAPI } from '../../../server/routes/create_apm_api';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
@elastic/apm-ui  We still have a lot of this to remove from the public directory, it might take some time to fix all since most of them are types which is automatically inferred.